tests/unit/source_code/test_jobs.py

```diff
@@ -103,21 +105,138 @@ def test_workflow_task_container_builds_dependency_graph_unknown_pypi_library(mo
     ws.assert_not_called()
 
 
-def test_workflow_task_container_builds_dependency_graph_for_python_wheel(mock_path_lookup, graph):
-    ws = create_autospec(WorkspaceClient)
-    ws.workspace.download.return_value = io.BytesIO(b"test")
+class TestWorkflowTaskContainerDependencyGraphForLibraries:
```

Comment:
every test is a top-level function everywhere, we don't use classes for grouping. refactor.

Reply:
Done.